### DISCUSSION TITLE
Componente de conteúdo avançado B

### DISCUSSION THREAD

**Discussion by vinrosa** [Telefonica/mistica-design#1196] [Proposals]
### Goal

Criar um componente de conteúdo ”avançado” que será utilizado no componente data card através da propriedade de slot e que possa ser reutilizável de diversas maneiras e por combinações diferentes.

### Context

Os cards apresentados no novo conceito para o novo app Vivo possuem elementos de conteúdo que não são suportados pelo atual componente data card, a favor de suportar esse conteúdo, e com um racional de criar o menor número possível de variações, concluímos que com 4 novos componente de conteúdo  para uso da Vivo atenderemos os casos atuais da aba de assinaturas.
Esse componente suporta os componente de Saldo

### Impact

Não gera impacto em componentes já existentes na biblioteca Mística.
Componente vai ser utilizado na lib de componentes avançados

### Discussion Questions

Alguma sugestão de melhoria?

### Additional considerations

Componentes relacionados:
https://github.com/Telefonica/mistica-design/discussions/1195
https://github.com/Telefonica/mistica-design/discussions/1197
https://github.com/Telefonica/mistica-design/discussions/1198

### Do you have a visual proposal?

![Comp B](https://user-images.githubusercontent.com/129896641/233357196-56e9cc37-3f7b-48d5-af96-881bd326d800.jpg)

**Comment by yceballost** [comment]
Creo que puede ser interesante tener un componente de precio en Mística para que se pueda gestionar de manera consistente el uso de los precios en un producto.

En algún momento se hablo sobre esto en esta discussion https://github.com/Telefonica/mistica-design/discussions/831

@vinrosa pensáis que sería posible hacer una definición para un componente precio que pueda funcionar transversalmente? podrían existir variantes de tamaños para diferentes casos
